packages/flutter/test/widgets/selectable_text_test.dart

```diff
@@ -505,34 +505,6 @@ void main() {
     expect(findRenderEditable(tester).textHeightBehavior, textHeightBehavior);
   });
 
-  testWidgets('Cursor blinks when showCursor is true', (WidgetTester tester) async {
```

Comment:
@chunhtai this change would make SelectableText lose its blinking cursor, does that sound ok to you?

Reply:
Changed to showCusor now the tests are passing.